src/python/pants/backend/go/sdk.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+import shlex
+import textwrap
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+from pants.backend.go.distribution import GoLangDistribution
+from pants.core.util_rules.external_tool import DownloadedExternalTool, ExternalToolRequest
+from pants.engine.fs import CreateDigest, Digest, FileContent, MergeDigests
+from pants.engine.internals.selectors import Get
+from pants.engine.platform import Platform
+from pants.engine.process import BashBinary, Process
+from pants.engine.rules import collect_rules, rule
+from pants.util.logging import LogLevel
+
+
+@dataclass(frozen=True)
+class SetupGoSdkProcess:
+    digest: Digest
+    command: Tuple[str, ...]
```

Comment:
In Python, this is `argv`. Feel free to diverge, but FYI.

Reply:
Gping to keep it this way for now. If we want to standardize on something, my vote is for `args`. `argv` is a C-ism and most people are not going to know what we called it `argv`.